examples/error.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://console.redhat.com/api/schemas/events/v1/events.json",
+  "id": "urn:redhat:console:event:5864ac25-4c52-4c87-bd28-9909a4fa3187",
+  "source": "urn:redhat:console:app:subscriptions",
+  "specversion": "1.0",
+  "type": "com.redhat.console.subscriptions.error",
+  "subject": "urn:redhat:console:org:org123",
+  "time": "2018-04-05T17:31:00Z",
+  "redhatorgid": "org123",
+  "redhatconsolebundle": "rhel",
+  "dataschema": "https://console.redhat.com/api/schemas/core/v1/error.json",
+  "data": {
+    "error": {
+      "code": "SUBSCRIPTIONS1004",
```

Comment:
Do these codes from from a shared lexicon or can people just make them up as needed? I'm fine w/ the latter.

Reply:
oops, just saw below that it's defined as string :) ignore this.